a-rmz/JEasyCrypto
# Feat/build

#### What does this PR do?
- Add a build script
- Enable CI with Travis

#### Where should the reviewer start?
`build.sh`

#### How should this be manually tested?
1. Create a new branch and push it.
2. Check the [status](https://travis-ci.org/a-rmz/JEasyCrypto) of the project.
3. :tada:

#### Any background context you want to provide?
To enforce good pull requests and quality features. :muscle:
